app/locale/af/LC_MESSAGES/django.po

```diff
@@ -695,6 +695,10 @@ msgstr "Vergelyk hierdie rekords"
 msgid "Contact information"
 msgstr "Kontakinligting"
 
+#, python-format
+msgid "Content purported to be compressed with %s but failed to decompress."
```

Comment:
I'm not sure exactly why, but it looks like messages from files in `app/vendors` is getting added to the django.po files. That's not necessary -- we only need our own messages in here.

Reply:
Even am trying to understand how messages are getting added to django.po files. Neither any of regular expression in my code is appending so. What exactly is the ask here ?